R/xgboost-deployment.R

```diff
@@ -7,16 +7,15 @@
 #' \item Push these predictions to SQL Server or CSV
 #' }
 #' @docType class
-#' @usage XGBoostDeployment(type, df, grainCol, testWindowCol, 
-#' predictedCol, impute, debug)
+#' @usage XGBoostDeployment(type, df, grainCol,
+#' predictedCol, impute, debug, cores, modelName)
 #' @import caret
 #' @import doParallel
 #' @import xgboost
 #' @importFrom R6 R6Class
 #' @param type The type of model (must be multiclass)
 #' @param df Dataframe whose columns are used for new predictions
 #' @param grainCol The dataframe's column that has IDs pertaining to the grain
-#' @param testWindowCol (Depreciated) Predictions will be made for all rows.
 #' @param predictedCol Column that you want to predict. If you're doing
 #' classification then this should be Y/N.
```

Comment:
It might be good to provide an example if there were more than 2 classes since the main purpose of XGBoost seems to be for multiclass (2 or more). The Y/N might not highlight the functionality as well.